addons/stock_account/models/res_company.py

```diff
@@ -0,0 +1,9 @@
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _name = "res.company"
+    _inherit = "res.company"
+
+    account_production_wip_account_id = fields.Many2one('account.account', string='Production WIP Account', check_company=True)
+    account_production_wip_overhead_account_id = fields.Many2one('account.account', string='Production WIP Overhead Account', check_company=True)
```

Comment:
do we need some kind of settings to select these accounts?

Reply:
It's in the enterprise branch :wink: since we only expect to be used when "accountant" is installed